docs/EXTENDED-ARCHIVES.md

```diff
@@ -0,0 +1,344 @@
+information about the image, the target device type, and the vendor, and
+references all the other files living inside the `.meta` directory. This file
+may contain the following top level properties:
+
+- `vendor`: Information about the image vendor
+- `device`: Information about the image device type
+- `image`: Information about the image itself
+
+For example:
+
+```json
+{
+  "vendor": { ... },
+  "device": { ... },
+  "image": { ... }
+}
+```
+
+Each of the top level properties are described in detail below.
+
+Vendor
+------
+
+> A vendor represents a manufacturer of a device type and potential publisher
+  of official images.
+
+_Some examples of vendors are: Beagleboard, Raspberry Pi
+Foundation, Intel, and Tizen._
+
+Here's an example of a real-world vendor manifest for Beagleboard:
+
+```json
+{
+  "name": "Beagleboard",
+  "url": "http://beagleboard.org",
+  "supportUrl": "http://beagleboard.org/support/",
+  "logoPath": "vendor.svg",
+  "colorScheme": {
+    "backgroundColor": "#535760",
+    "foregroundColor": "#FFFFFF",
+    "primaryColor": "#5793db"
+  }
+}
+```
+
+A vendor may be described with any of the following properties:
+
+### `name (String)`
+
+The display human-friendly name of the vendor.
+
+### `url (String)`
+
+The main url of the vendor, usually the landing page.
+
+### `supportUrl (String)`
+
+The vendor url where users can get general support.
+
+This could be a link to a forum, IRC room, troubleshooting page, support form,
+etc.
+
+### `logoPath (String)`
+
+The path to the vendor's logo in SVG format.
+
+The SVG format requirement ensures that the logo looks good in all types of
+screen resolutions.
+
+### `colorScheme (Object)`
+
+The vendor specific color scheme, if any.
+
+Declaring a vendor-specific color scheme allows a certain degree of branding to
+be applied to the application on the fly as soon as a vendored image is
+selected.
+
+You may declare the following colors, in hexadecimal format:
+
+- `backgroundColor`: The application background color
+- `foregroundColor`: The application foreground color
+- `primaryColor`: The application primary color
+
+Device
+------
+
+> A device represents a specific device type manufactured by a certain vendor.
+
+_Some examples of devices are: Beaglebone Black, Beaglebone Green, Intel Edison,
+and Raspberry Pi 3._
+
+Here's an example of a real-world device manifest for Raspberry Pi 3:
+
+```json
+{
+  "displayName": "Raspberry Pi 3",
+  "arch": "armv7hf",
+  "slug": "raspberrypi3",
+  "slugAliases": [
+    "raspberry-pi3",
+    "rpi3"
+  ],
+  "url": "https://www.raspberrypi.org/products/raspberry-pi-3-model-b/",
+  "photoPath": "raspberry-pi-3-photo.jpg"
+}
+```
+
+A device may be described with any of the following properties:
+
+### `displayName (String)`
+
+The human-friendly name of the device.
+
+### `arch (String[])`
+
+The device architecture.
+
+The current possible values are: `aarch64`, `amd64`, `armv5e`, `armv7hf`,
+`i386`, and `rpi`.
```

Comment:
how arches are named is a massive chaos. One thing is for sure, there will be one canonical name, but a bunch of alternatives for each. no way around that. Not much to comment on the specifics here, leaving it to @jviotti